Source/WebCore/html/ImageBitmap.cpp

```diff
@@ -779,9 +778,11 @@ class PendingImageBitmap final : public ActiveDOMObject, public FileReaderLoader
 
     void createImageBitmapAndCallCompletionHandlerSoon(RefPtr<ArrayBuffer>&& arrayBuffer)
     {
-        ASSERT(!m_createImageBitmapTimer.isActive());
         m_arrayBufferToProcess = WTFMove(arrayBuffer);
-        m_createImageBitmapTimer.startOneShot(0_s);
+        scriptExecutionContext()->eventLoop().queueTask(TaskSource::InternalAsyncTask, [weakThis = WeakPtr { *this }]() {
```

Comment:
Nit - I think you can omit `()` here.

(And in a few other places in this patch)

Reply:
Will remove.